ui/l10n/en.js

```diff
@@ -19,7 +19,7 @@ var dictionary = {"ICMP.code":"ICMP Code",
 "changed.item.properties":"Changed item properties",
 "confirm.enable.s3":"Please fill in the following information to enable support for S3-backed Secondary Storage",
 "confirm.enable.swift":"Please fill in the following information to enable support for Swift",
-"error.could.not.change.your.password.because.ldap.is.enabled":"Error could not change your password because LDAP is enabled.",
+"error.could.not.change.your.password.because.non.native.user":"Error could not change your password because user is not a native CloudStack user.",
```

Comment:
@resmo this is changed here ^^ for english translation, for others I won't know exactly what string/text to use in respective languages.